contracts/gateway.fc

```diff
@@ -344,3 +355,31 @@ int seqno() method_id {
 
     return state::seqno;
 }
+
+;; returns tx fee for the specified operation
+int calculate_gas_fee(int op) method_id {
```

Comment:
It seems like this function is not used in the code?

Reply:
it's a getter function that might be used via RPC to estimate tx fee in TON